### DISCUSSION TITLE
Modal opened multiple times at once

### DISCUSSION THREAD

**Discussion by arsrobota** [Orange-OpenSource/Orange-Boosted-Bootstrap#2653] [Q&A]
Hello,
I'm integrating Boosted-Boostrap in a Symfony web app through epo/theme-boosted-ui-bundle.
Each time a modal (but also other components such as dropdowns) opens up, opening is triggered thre times (by js/dist/modal.js, dist/js/boosted.js and dist/js/boosted.esm.js). Div element with modal class itself is duplicated but the show.bs.modal event is raised three times and so is grey backdrop behind modal.
How to avoid this ?
Thanks by advance

**Comment by louismaximepiton** [comment]
Hello,
Can you maybe join a minimum reproducible example with your request so we can start debugging it ? You can use for instance [StackBlitz](https://stackblitz.com/) or [CodePen](https://codepen.io/).
Just FYI, you should import either the bundle or the esm version or via custom imports as mentioned in our [JavaScript page](https://boosted.orange.com/docs/5.3/getting-started/javascript/).
Hope this is helpful.

**Comment by arsrobota** [comment]
No bug, no problem. My bad, I forgot to remove explicite calls to Boosted library.
